core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -667,7 +667,10 @@ public String toString() {
               "${%s}/core-site.xml:${%s}/hdfs-site.xml", Name.CONF_DIR, Name.CONF_DIR))
           .setDescription("Location of the HDFS configuration file to overwrite "
               + "the default HDFS client configuration. Note that, these files must be available"
-              + "on every node.")
+              + "on every node, but if there are a "
```

Comment:
why not just support this behavior by default? I think it will be strictly better

Reply:
I see. So, this PR achieves this behavior by reading in the conf files, and converting them to mount options, so when workers get the ufs config, they will get all the hadoop configs too.

This is turned on when "kv@" is prefixed. Are you saying just enable this behavior all the time?

Reply:
@apc999 @gpang  It can be more simply if we turned on "kv@" all the time, but not sure wether we should support different worker use different hdfs config files? If not, I'm ok to push a new commit to strictly support this behavior only.

Reply:
I push the last commit  c344662c117aa0d9401ea2e8d9098f39d0699f70